### DISCUSSION TITLE
goldendict-ng软件搜索栏及标签页藏文显示有误。

### DISCUSSION THREAD

**Discussion by baxink** [xiaoyifang/goldendict-ng#1405] [Q&A]
<img width="952" alt="GoldenDict软件中藏文显示问题" src="https://github.com/xiaoyifang/goldendict-ng/assets/32429912/45d13286-7503-433f-b003-0457165e6415">
你好，在使用这个goldendict-ng版时发现搜索栏及标签页的藏文支持有问题，具体请见截图。同时为作对比，在截图中用MDict搜索栏做参考，显示正常。烦请告知修复方法。

**Comment by xiaoyifang** [comment]
版本提供下，我记得这个问题处理过了。

**Reply by baxink** [reply]
你好，版本请见附件图片。
<img width="574" alt="Snipaste_2024-02-27_09-17-52" src="https://github.com/xiaoyifang/goldendict-ng/assets/32429912/2c13430e-430a-46f8-b13e-e36cb13bb302">

**Comment by xiaoyifang** [comment]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/9bfcfe2a-8509-4a5f-9025-4dd90afbbbf4)

换个支持藏文的字体

**Reply by baxink** [reply]
设置成功了，谢谢！
